spec/relaton_ccsds/bibliography_spec.rb

```diff
@@ -1,17 +1,36 @@
-describe RelatonCcsds::Bibliography do
+class RelatonCcsds::TestHitCollection < RelatonCcsds::HitCollection
+  # override default index method to avoid index downloading
```

Comment:
For integration tests we need to force downloading index, not disable. You can see before block below that intentionally forces to download index.

Reply:
@andrew2net we want to have fixed input data, so our tests will not stop working after index updates